camel/agents/tool_agents/hugging_face_tool_agent.py

```diff
@@ -50,96 +50,96 @@ def __init__(
+
+        # If you'd like to keep a state across executions or to pass non-text objects to the agent, 
+        # you can do so by specifying variables that you would like the agent to use. For example,
+        # you could generate the first image of rivers and lakes, and ask the model to update that picture to add an island by doing the following:
+        picture = {self.name}.step("Generate a picture of rivers and lakes.")
+        picture.save("./picture.png")
+        updated_picture = {self.name}.step("Transform the image in `picture` to add an island to it.", picture=picture)
+        updated_picture.save("./updated_picture.png")
+
+        capybara_sea_image = {self.name}.step("Draw me a picture of the `prompt`", prompt="a capybara swimming in the sea")
+        capybara_sea_image.save("./capybara_sea_image.png")
+
+        # Document question answering
+        answer = {self.name}.step(
+            "In the following `document`, where will the TRRF Scientific Advisory Council Meeting take place?",
+            document=document,
+        )
+        print(answer)
+
+
+        # Text to image
+        boat_image = {self.name}.step("Generate an image of a boat in the water")
+        boat_image.save("./boat_image.png")
+
+        # Unconditional image captioning
+        boat_image_caption = {self.name}.step("Can you caption the `boat_image`?", boat_image=boat_image)
+        print(boat_image_caption)
+
+        # Text to image -> Unconditional image captioning -> Text to speech
+        boat_audio = {self.name}.step("Can you generate an image of a boat? Please read out loud the contents of the image afterwards")
+
+        # Text downloading
+        document = {self.name}.step("Download the text from http://hf.co")
+        print(document)
+
+        # Text summarization
+        summary = {self.name}.step("Summarize the following text: `document`", document=document)
+        print(summary)
+
+        # Text downloading -> Text summarization -> Text to speech
+        audio = {self.name}.step("Read out loud the summary of http://hf.co")
+        ```
+
+        Chat-based execution (chat), the agent also has a chat-based approach, using the chat() method:
+        ```
+        # Clean the chat history
+        {self.name}.reset()
+
+        # Text to image
+        capybara_image = {self.name}.chat("Show me an an image of a capybara")
+        capybara_image.save("./capybara_image.png")
+
+        # Image transformation
+        transformed_capybara_image = {self.name}.chat("Transform the image so that it snows")
+        transformed_capybara_image.save("./transformed_capybara_image.png")
+
+        # Image segmentation
+        segmented_transformed_capybara_image = {self.name}.chat("Show me a mask of the snowy capybaras")
+        segmented_transformed_capybara_image.save("./segmented_transformed_capybara_image.png")
+        ```
+        """
+        super(HuggingFaceToolAgent, self).__init__(name, description)
```

Comment:
Sorry that the original indentation likes ugly. But it may be good to keep it since it does not introduce extra white spaces for the prompt.

Reply:
I see, will fix that.